client/van_connector_create_test.go

```diff
@@ -20,8 +21,8 @@ func TestConnectorCreateError(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	// Create the client
-	cli, err := newMockClient("skupper", "", "")
+	cli, err := getVanClient(testing.Short(), "namespace")
```

Comment:
As there will be additional checks against the clusterRun flag to control test customization, checks, etc, I would lean towards not encapsulating client setup in getVanClient and keep it inline for the test now:

// Create the client
var cli *VanClient
var err error
if !clusterRun {
	cli, err = newMockClient("my-namespace", "", "")
} else {
	cli, err = NewClient("my-namespace", "", "")
}
assert.Check(t, err, "doc text")

Also, testing.Short() is a testing flag which we may use  at some point but the flag being introduced in this pr is for the purpose of selecting mock or cluster client.

Reply:
Done!